src/aks-preview/HISTORY.md

```diff
@@ -20,6 +20,11 @@ Release History
 * Add support for nodepool adding with Capacity Reservation Group.
   * `az aks nodepool add --crg-id`
 
+0.5.53
+++++++
+
+* Add `--message-of-the-day` flag for `az aks create` and `az aks nodepool add` for Linux message of the day.
+
```

Comment:
Need to update the version. Maybe 0.5.56? Please also update `setup.py`.

Reply:
yep. also still reworking the encoding, not quite done yet.